acs-core/src/main/java/com/arrow/acs/client/model/ProductFeatureModel.java

```diff
@@ -0,0 +1,48 @@
+package com.arrow.acs.client.model;
+
+public class ProductFeatureModel {
```

Comment:
according to document from story, the name of this class should be" ProductFeature"